drawing.go

```diff
@@ -513,13 +513,18 @@ func (f *File) drawBaseChart(formatSet *formatChart) *cPlotArea {
 // drawDoughnutChart provides a function to draw the c:plotArea element for
 // doughnut chart by given format sets.
 func (f *File) drawDoughnutChart(formatSet *formatChart) *cPlotArea {
```

Comment:
Please add unit test for this change at [`chart_test.go`](https://github.com/xuri/excelize/blob/56aa6b82637b3210be470a8ebac1fdec2b2a6a30/chart_test.go#L140).